src/item_location.cpp

```diff
@@ -81,50 +119,33 @@ class item_location::impl
         }
 
         virtual tripoint position() const {
```

Comment:
Looks like this and other functions in `item_location::impl::nowhere` need to be marked as `override`